roles/tarzan/README.md

```diff
@@ -23,11 +23,13 @@ Set `tarzan_singularity_image_sha1` in `defaults/main.yml` to this value.
 
 It is recommended to have separate certificates for staging and production.
 
-On the node running Kong, run: `openssl req -x509 -newkey rsa:2048 -keyout tarzan_key.pem -out tarzan_cert.pem -days 1460 -nodes -subj '/CN=miarka2.uppmax.uu.se'`. (Or `CN=miarka1.uppmax.uu.se` for production.)
+On the node running Kong, run:
+```
+openssl req -config <tarzan_conf_dir>/tarzan_cert_config.txt -newkey rsa:2048 -nodes -keyout tarzan_key.pem -x509 -days 1460 -out tarzan_cert.pem
+```
 
 Move the generated certificate and key to the paths defined by `tarzan_cert_file` and `tarzan_key_file` in `env_vars/site_[site]_env_[environment].yml`.
```

Comment:
actually, it seems like these paths are defined in `env_vars/site_upps_env_all.yml` and they are dynamically resolved att playbook runtime..

Reply:
Ah good point, yeah, maybe we can keep it for now to get this merged :)